lib/segment/src/segment.rs

```diff
@@ -618,6 +623,19 @@ impl Segment {
             .collect()
     }
 
+    pub fn read_by_id_stream(
```

Comment:
Is this function used?

Reply:
Thanks for noticing, I meant to extract it from `filtered_read()`